src/services/item/plugins/importExport/service.ts

```diff
@@ -220,10 +222,9 @@ export class ImportExportService {
         const { mimetype } =
           (item.extra[ItemType.S3_FILE] as S3FileItemExtra) ||
           (item.extra[ItemType.LOCAL_FILE] as LocalFileItemExtra);
-        const fileStream = await this.fileItemService.download(actor, repositories, {
-          fileStorage,
+        const url = (await this.fileItemService.download(actor, repositories, {
           itemId: item.id,
-        });
+        })) as string;
```

Comment:
Casting necessary because the method `download` is quite a mess. To be handled in another PR.